src/middleware/register-context.js

```diff
@@ -0,0 +1,12 @@
+import { asValue } from 'awilix'
+
+// Register Context helps to add request-specific data to the scope.
+export const registerContext = async (ctx, next) => {
```

Comment:
Oh, just noticed the other middleware is declared as a regular async function, and also uses docblock style comments, 

```
/**
 * Like this
 */
```

Reply:
Should I change the other middleware?

Reply:
Oh snap, I was too quick to merge! Preferably change this one to use function, arrow funcs are usually used inside functions, while functions are the top-level (they have no `this` binding anyway)